client/src/component/NavBar/NavBar.jsx

```diff
@@ -1,32 +1,72 @@
-import React, {useState, useEffect} from "react";
-import {useSelector} from 'react-redux'
+import React from "react";
+import { useSelector } from "react-redux";
+import { theStore } from "../../index";
 
-import { Link, Nav, Ul } from "../Styles/navStyles";
+import { Link, Nav, Ul, Button } from "../Styles/navStyles";
 
 const NavBar = () => {
   const user = useSelector(state => state.user);
 
+  const handleLogout = async e => {
+    // await Axios.post(`${endpoint}/logout`);
+    theStore.dispatch({
+      type: "user logged out"
+    });
+    window.location.replace("/");
```

Comment:
Looks like we're redirecting them to the marketing landing page. Makes sense

Reply:
Yeah I left it has a window redirect as it will stand as a double check to make sure the global state is cleared